src/main/java/com/prgrms/commandLineApplication/voucher/discount/FixedDiscount.java

```diff
@@ -0,0 +1,32 @@
+package com.prgrms.commandLineApplication.voucher.discount;
+
+import com.prgrms.commandLineApplication.validator.Validator;
+import com.prgrms.commandLineApplication.voucher.VoucherType;
+
+public class FixedDiscount extends Discount {
+
+  private final int discountAmount;
+  public static final int MINIMUM_VALUE = 0;
+  public static final int MAXIMUM_VALUE = 10_000;
```

Comment:
[nit]
```suggestion
  public static final int MINIMUM_VALUE = 0;
  public static final int MAXIMUM_VALUE = 10_000;
  
  private final int discountAmount;
```
상수, 불변 멤버변수, 멤버변수 순으로 선언하는 걸 선호해요!

Reply:
abfb330